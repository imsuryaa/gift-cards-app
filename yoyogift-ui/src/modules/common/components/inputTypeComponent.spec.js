import React from "react";
import { mount } from "enzyme";

import InputTypeComponent from "./inputTypeComponent";

describe("Footer Test Suite", () => {
  it("should render expected HTML", () => {
    //Mocks
    let classes = { root: { marginTop: "2px" } }
    expect(mount(<InputTypeComponent />).html()).toMatchSnapshot();
  });
});