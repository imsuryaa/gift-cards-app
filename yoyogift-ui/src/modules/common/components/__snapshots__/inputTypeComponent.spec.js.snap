// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Test Suite should render expected HTML 1`] = `"<div class=\\"MuiFormControl-root-1\\"><label class=\\"MuiFormLabel-root-16 MuiInputLabel-root-5 MuiInputLabel-formControl-10 MuiInputLabel-animated-13 MuiInputLabel-shrink-12\\" data-shrink=\\"true\\"></label><div class=\\"MuiInputBase-root-36 MuiInput-root-23 MuiInput-underline-27 MuiInputBase-formControl-37 MuiInput-formControl-24 MuiInputBase-adornedStart-40 MuiInputBase-adornedEnd-41\\"><input aria-invalid=\\"false\\" class=\\"MuiInputBase-input-46 MuiInput-input-31 MuiInputBase-inputAdornedStart-51 MuiInputBase-inputAdornedEnd-52\\" type=\\"text\\" value=\\"\\"></div><p class=\\"MuiFormHelperText-root-53\\">  </p></div>"`;
